stdlib/libblastrampoline_jll/Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "libblastrampoline_jll"
 uuid = "8e850b90-86db-534c-a0d3-1478176c7d93"
-version = "5.4.0+0"
+version = "5.7.0+0"
 
 [deps]
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 
 [compat]
-julia = "1.8"
+julia = "1.10"
```

Comment:
We should make this 1.9, so that it can be backported and made available in 1.9.1.

Reply:
When we backport, we can set it accordingly.  I don't want to set it to `1.9`, and then it's only available in `1.9.1`